### DISCUSSION TITLE
Black screen and not loading anything

### DISCUSSION THREAD

**Discussion by pablodiegoss** [icosa-gallery/open-brush#777] [Q&A]
Hello!

I have an HTC Vive VR system, tried opening google brush which was on my steam library, the game loaded and all I could see was a black infinite background and my controllers had a white glow with two red big X's on my thumb pads. I thought I the google brush had some problem, then I discovered OpenBrush on the steam community tab.

However, Openbrush gave me the same output, black void with two red X's on my controllers and nothing more, no errors, no warnings. 

How can I debug this to see whats wrong? Is my vive system not supported anymore? I used to open google brush normally to play around but its been a couple years since I had to move and sell my computer, I just bought a new one and my vive is the same though.

Other games are normal.

**Comment by andybak** [comment]
Can you check what your default OpenXR runtime is set to in the SteamVR settings: 
![image](https://github.com/user-attachments/assets/f3be95d0-b94b-4c34-b227-d0617f2e102b)

**Comment by pablodiegoss** [comment]
It shows as SteamVR but no OpenXR API Layers, not even sure what that means. Meta is also off.

![image](https://github.com/user-attachments/assets/132e73cc-001b-4701-b1e0-0b48bc346fda)

**Comment by andybak** [comment]
@mikeskydev - any ideas?
